config/src/test/scala/com/typesafe/config/impl/ConfigSubstitutionTest.scala

```diff
@@ -924,7 +924,7 @@ class ConfigSubstitutionTest extends TestUtils {
 
     @Test
     def substSelfReferenceIndirect() {
-        val obj = parseObject("""a=1, b=${a}, a=${b}""")
+        val obj = parseObject("""b=${a}, a=${b}""")
```

Comment:
Thanks. I didn't know from the test name if it was supposed to be that you could have a key twice. It'd probably be good to have one test for testing indirect self test and one test for having a key twice